p3/management/commands/talk_abstracts.py

```diff
@@ -73,10 +73,10 @@ def handle(self, *args, **options):
 
                 schedule = talk_schedule(talk)
                 if not schedule and verbose:
-                    print('ERROR: Talk {} is not scheduled.'.format(talk))
+                    self.stdout.write('ERROR: Talk {} is not scheduled.'.format(talk))
```

Comment:
log.error example

Reply:
I don't think it's a good idea in this case - an incorrect django config could mean the logs are only saved to a file on disk and are never shown to the user. I agree that lines like this should not write to stdout - I'll update them so that they write to stderr instead.